PortScanner.py

```diff
@@ -1,5 +1,6 @@
 import optparse
 import socket
+

```

Comment:
Why extra line?

Reply:
import block should be separated from the code

Reply:
Aaeh may be...not interested at this time ...so cancelling your request only for fun.. :-)